test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -714,7 +719,17 @@ def test_qconv2d_cpu(self):
         r"""
         This testcase will quantize a single Conv2d module.
         """
-        self._qconv2d_cpu_test_helper()
+        self._qconv2d_cpu_test_helper("cpu")
+
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    @skipIfRocm
+    @skipIfNoXPU
+    def test_qconv2d_xpu(self):
+        r"""
+        This testcase will quantize a single Conv2d module.
+        """
+        self._qconv2d_cpu_test_helper("xpu")
```

Comment:
This function is called cpu-specific but accepts an 'xpu' as input. Sound wired.
In my understanding, these UTs are used to test the pattern matcher. We don't have any other UT that could test the accuracy of qconv, right?

Reply:
sure, let me change the ut name of these helper functions.

Reply:
the function name is changed to `_qconv_test_helper`

Reply:
Have you changed it?